### PR TITLE
Images should be searched relative to the manifest file on cloud storage

### PR DESCRIPTION
**Pull request by Marishka17**
### How has this been tested?

Tests have been added
### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly~~
- [x] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by kirill-sizov** [comment]
@Marishka17 Try to merge new changes from develop branch to enable PR workflows for your PR

File: tests/rest_api/test_tasks.py [hunk 1]
```diff
@@ -35,6 +35,12 @@ def generate_image_files(count):
 
     return images
 
+def get_cloud_storage_content(username, cloud_storage_id, manifest):
+    with make_api_client(username) as api_client:
+        (_, response) = api_client.cloud_storages_api.cloudstorages_retrieve_content(cloud_storage_id, manifest_path=manifest)
```
Comment (zhiltsov-max) [inline tests/rest_api/test_tasks.py:40]: Please check why this operation is called in full name here. Probably, the leading part should be removed by the postprocessing script: `cloudstorages_retrieve_content` -> `cloudstorages.retrieve_content`

**Comment by kirill-sizov** [comment]
The failed test not related with this PR changes, so PR can be considered as green

**Review by zhiltsov-max — APPROVED** [review]

**Comment by nmanovic** [comment]
@Marishka17 , @kirill-sizov , should we cleanup archive data.tar.bz2 somehow? Now it can have some temporary files

image

**Comment by nmanovic** [comment]
```
input_value = ['image_case_65_1.png', 'image_case_65_2.png'], valid_classes = (<class 'dict'>,), path_to_item = ['received_data']
configuration = <cvat_sdk.configuration.Configuration object at 0x116e1bcd0>, spec_property_namin

...

>               raise get_type_error(input_value, path_to_item, valid_classes, key_type=key_type)
E               cvat_sdk.exceptions.ApiTypeError: Invalid type for variable 'received_data'. Required value type is dict and passed type was list at ['received_data']

.env/lib/python3.8/site-packages/cvat_sdk/model_utils.py:1401: ApiTypeError

```


```
=============================================================================== short test summary info ===============================================================================
FAILED tests/rest_api/test_tasks.py::TestPostTaskData::test_create_task_with_cloud_storage_files[1-manifest.jsonl-] - cvat_sdk.exceptions.ApiTypeError: Invalid type for variable 'r...
FAILED tests/rest_api/test_tasks.py::TestPostTaskData::test_create_task_with_cloud_storage_files[2-sub/manifest.jsonl-org2] - cvat_sdk.exceptions.ApiTypeError: Invalid type for var...
====================================================================== 2 failed, 305 passed in 365.58s (0:06:05) ======================================================================
```

File: cvat/apps/engine/views.py [hunk 2]
```diff
@@ -1895,7 +1895,7 @@ def create(self, request, *args, **kwargs):
                 location=OpenApiParameter.QUERY, type=OpenApiTypes.STR),
         ],
         responses={
-            '200': OpenApiResponse(response=OpenApiTypes.OBJECT, description='A manifest content'),
+            '200': OpenApiResponse(response=OpenApiTypes.BINARY, description='A manifest content'),
```
Comment (nmanovic) [inline cvat/apps/engine/views.py:1898]: @Marishka17 , is the type correct? It looks like the code should return a list of strings.

**Review by nmanovic — APPROVED** [review]
LGTM

**Pull request merged**